integreat_cms/core/signals/feedback_signals.py

```diff
@@ -63,7 +63,12 @@ def feedback_create_handler(sender: ModelBase, **kwargs: Any) -> None:
     :param sender: The class of the feedback that was deleted
     :param \**kwargs: The supplied keyword arguments
     """
-    if (instance := kwargs.get("instance")) and (
-        feedback_ptr := getattr(instance, "feedback_ptr", None)
-    ):
-        invalidate_obj(feedback_ptr)
+    try:
+        if (instance := kwargs.get("instance")) and (
+            feedback_ptr := getattr(instance, "feedback_ptr", None)
+        ):
+            invalidate_obj(feedback_ptr)
+    except Feedback.DoesNotExist:
+        # When installing fixtures, the related feedback might not yet be initialized.
+        # In that case cache invalidation is not necessary though
+        pass
```

Comment:
No, this code was originally part of #2781, but did not end up being included for merging, because it was not required anymore. This exception cannot occur anymore and the code should be removed from develop imo

Reply:
Oh. Could you please open up a separate PR for that? I had assumed this was just a clean-up

Reply:
#3038 :smile: